python/tools/dht/tests.py

```diff
@@ -17,6 +17,15 @@
 import numpy as np
 import matplotlib.pyplot as plt
 import networkx as nx
+import subprocess
+import re
+
+from matplotlib.ticker import FuncFormatter
+import math
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.ticker import FuncFormatter
```

Comment:
This diff shows duplicate imports. Probably happenned while resolving conflicts. You should revert this change (lines 20-28).